src/SDKs/CognitiveServices/dataPlane/Search/BingVisualSearch/BingVisualSearch/Generated/SdkInfo_VisualSearchAPI.cs

```diff
@@ -0,0 +1,18 @@
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
```

Comment:
Please run `msbuild build.proj /t:build /p:Scope=SDKs\/CognitiveServices/dataPlane/Search/BingVisualSearch/` and commit the `.props` file generated

Reply:
hmm, i see a .props in the /obj output on building--should i move that file somewhere and commit it?

Reply:
@shakao 
Talked to @shahabhijeet I think we can skip the .props file for now and we will investigate later why the script might have failed (the `.props` file is more important for the management sdks)